commands/acl-list.md

```diff
@@ -1,6 +1,6 @@
-The command shows the currently active ACL rules in the Redis server. Each
+The command shows the currently active ACL rules in the Valkey server. Each
 line in the returned array defines a different user, and the format is the
-same used in the redis.conf file or the external ACL file, so you can
+same used in the valkey.conf file or the external ACL file, so you can
```

Comment:
I don't remember what we called this. maybe it's not yet decided?

Please check before taking my suggestion.

```suggestion
same used in the server.conf file or the external ACL file, so you can
```